config/locales/views/comments/fr.yml

```diff
@@ -118,7 +118,8 @@ fr:
           aria_label: Report %{user}'s comment as abusive or violating our code of conduct and/or terms and conditions
       footer:
         heart:
-          title: heart
+          aria_label: like
+          title: Like
```

Comment:
Same with this one - do we need it any more?

Reply:
These are also used by the same files mentioned above